cec-mini-kb.cpp

```diff
@@ -173,23 +173,37 @@ void uinput_dev_deinit(void){
 	}
 }
 
+bool legal_int(char *str)
+{
+    char* end;
+    long int num = std::strtol(str, &end, 10);
+
+    if (*end == '\0') {
+    	return true;
+    }
+    else {
+        return false;
+    }
+}
+
```

Comment:
This does not guarantee that value will be within atoi range, which is only `int`.
Anyway this function can be dropped, easier test below.

Reply:
:+1: